edx_rest_api_client/client.py

```diff
@@ -1,17 +1,16 @@
 import datetime
 import json
-import os
 import socket
+import os
 
 import crum
 import requests
 import requests.utils
-import slumber
```

Comment:
- Hopefully `slumber` can be fully removed from this library, including the requirements files. That means removing from the `*.in` requirement files, and then running `make upgrade` in the top-level directory to generate the updated `*.txt` requirement files. 
- It looks like slumber is also used in an exceptions file: https://github.com/search?q=repo%3Aopenedx%2Fedx-rest-api-client%20slumber&type=code
- You still need to do a major upgrade of the version to 6.0.0 here: https://github.com/openedx/edx-rest-api-client/blob/6bbb88e1325d3adc7efe9015bee004411b18780e/edx_rest_api_client/__version__.py#L1, and update the CHANGELOG.rst. See https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0051-bp-conventional-commits.html for handling backward incompatible changes like this one.

Reply:
@robrap Yes I need to do a version change and then add something to the CHANGELOG indicating that we are removing the old client and that this is a breaking change:

Reply:
@Yagnesh1998: A reminder to update CHANGELOG.rst. See https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0051-bp-conventional-commits.html for handling backward incompatible changes like this one.